pkg/cephmgr/osd/device.go

```diff
@@ -294,6 +342,24 @@ func getDiskSize(context *clusterd.Context, name string) (uint64, error) {
 	return 0, fmt.Errorf("device %s not found", name)
 }
 
+func waitForPath(path string, executor exec.Executor) error {
```

Comment:
after we create partition on the device, we need to wait for `udev` to surface the `/dev/disk/by-partuuid` sym link.  There's a slight delay from creating the partition to that symlink showing up.